content/en/docs/userguide/install_waypoint.md

```diff
@@ -116,7 +121,25 @@ namespace default labeled with "istio.io/use-waypoint: waypoint"
 
 Then any requests from any pods using the Kmesh, to any service running in `default` namespace, will be routed through that waypoint for L7 processing and policy enforcement.
 
-#### Use waypoint at pod granularity:
+#### Install waypoint per service:
```

Comment:
Here we repeat the above. We can directly indicate that we can insall three granularity waypoints: ns , service and pod, under `6. Deploy waypoint`, and then start with `Install waypoint per service` as the title